CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# docs: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+* @Michsior14 @wheinze
```

Comment:
Don't we want to go with @igusdev/tools instead?

Reply:
Planned to do so once we have reached a beta state of this new setup. What do you think?

Reply:
Fine for me ;)